coregrind/vgdb-invoker-ptrace.c

```diff
@@ -41,7 +41,8 @@
 #include <sys/wait.h>
 
 #if defined(VGA_riscv64)
-/* TODO */
+/* By now kernel 6.0-rc3, this is the only way to get/set user regs for riscv. */
+#define USE_PTRACE_GETREGSET
```

Comment:
I think it would be better to reuse the definition of `USE_PTRACE_GETREGSET` on line 68, basically changing the code there to `#if defined(VGA_arm64) || defined(VGA_riscv64)`. This way the file would contain only one definition and it is slightly clearer which architectures use it.